spec/21-http_response_header_format.md

```diff
@@ -0,0 +1,184 @@
+# Trace Context HTTP Response Headers Format
+
+This section describes the binding of the distributed trace context to the `tracemeta` HTTP header.
+
+## Tracemeta Header
+
+The `tracemeta` HTTP response header field identifies a completed request in a tracing system. It has four fields:
+
+* `version`
+* `trace-id`
+* `parent-id`
+* `trace-flags`
+
+### Header Name
+
+Header name: `tracemeta`
+
+In order to increase interoperability across multiple protocols and encourage successful integration, by default vendors SHOULD keep the header name lowercase. The header name is a single word without any delimiters, for example, a hyphen (`-`).
```

Comment:
I'd like to remove this phrase: :
> by default vendors SHOULD keep the header name lowercase

The _by default_ part makes me think that this functionality maybe should be configurable, which doesn't make sense to me.  
And I'm not sure what the verb _keep_ means in this context.  

```suggestion
The header name is a single word without any delimiters, for example, a hyphen (`-`).
```

Also, this is described more clearly two lines down: 
> Vendors MUST expect the header name in any case (upper, lower, mixed), and SHOULD send the header name in lowercase.

Reply:
This wording was copied directly from the forward header spec:

traceparent: https://github.com/w3c/trace-context/blob/e32fa09fb7c2796ac2d10f5bb73cae249c4ae21b/spec/20-http_header_format.md#header-name
tracestate: https://github.com/w3c/trace-context/blob/e32fa09fb7c2796ac2d10f5bb73cae249c4ae21b/spec/20-http_header_format.md#header-name-1

Should it be changed in all three places?

Reply:
I don't know whether it makes sense to change it in all three places.  It doesn't feel like the highest priority.  

I'm OK with us leaving the wording as is.

Reply:
What about this?

```suggestion
In order to increase interoperability across multiple protocols and encourage successful integration, the header name SHOULD be lowercase. The header name is a single word without any delimiters, for example, a hyphen (`-`).
```

Reply:
👍